### PR TITLE
Doc mac

### PR DESCRIPTION
**Pull request by erinboyd**
Pull request to update Mac documentation for dev setup

### CODE REVIEW

**Comment by Fryguy** [comment]
@erinboyd This PR is not mergable...Please rebase.

**Comment by jrafanie** [comment]
@erinboyd Please rebase, thanks!

**Comment by erinboyd** [comment]
okay!

----- Original Message -----

**Comment by pilhuhn** [comment]
I think that could be closed now with #90 being in.

**Comment by kbrock** [comment]
maybe keep docs for chruby in addition to rvm?

`rvm` is great if you like that kind of thing.
some prefer something less invasive

**Comment by chrisarcand** [comment]
Might be stealing the thread here, but IMHO I think this doc is already a bit too large, detailing _a lot_ of things that could be easily outsourced to helpful documentation outside of this document. 

The detailed configuration of Postgres makes sense to me, but things like GitHub and installing Ruby? I like the [current doc on Ruby](https://github.com/ManageIQ/guides/blob/master/developer_setup.md#install-ruby) which points you to documentation on how to install such things - and those docs update themselves, not needing to be changed in this doc later. The amount of detail for generics in this document actually makes it kind of hard to read :confused: 

In any case, this PR is from March and quite outdated; the Ruby install docs and the Ruby version itself have been changed since, so I digress. Maybe time just to close this?

**Comment by pilhuhn** [comment]
I agree that the git&github section may be overkill here, as not everyone that tries to install ManageIQ from source will want to modify it.

File: developer_setup.md [hunk 1]
```diff
@@ -47,6 +47,15 @@
...
 
+* Configure memcached
+
+ ```bash
+  $ mkdir ~/Library/LaunchAgents
+  $ cp /usr/local/Cellar/memcached/$version/homebrew.mxcl.memcached.plist ~/Library/LaunchAgents/
+  $ launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.memcached.plist.
+  $ /opt/local/bin/memcached -d -p 11211
+  ```
+
```
Comment (kbrock) [inline developer_setup.md]: this is great

File: developer_setup.md [hunk 2]
```diff
@@ -56,8 +65,12 @@
   launchctl load ~/Library/LaunchAgents/homebrew.mxcl.postgresql.plist
 
   psql -c "CREATE ROLE root SUPERUSER LOGIN PASSWORD 'smartvm'"
+  psql -c "alter database vmdb_development owner to root”
+  
+  env ARCHFLAGS="-arch x86_64" gem install pg
```
Comment (kbrock) [inline developer_setup.md]: maybe move down to trouble shooting section?

File: developer_setup.md [hunk 3]
```diff
@@ -93,9 +106,16 @@
 
 ## Install Ruby
 
-* chruby: <https://github.com/postmodern/chruby>
```
Comment (kbrock) [inline developer_setup.md]: adding rvm information is great, but please don't remember `chruby`

**Comment by kbrock** [comment]
@erinboyd you want to rebase and we can get this in?

**Pull request closed without merging**